.github/workflows/release.yml

```diff
@@ -0,0 +1,29 @@
+name: release
+
+on:
+  push:
+    # run only against tags
+    tags:
+      - 'v*'
```

Comment:
Since we are not tagging just yet, should we have this continuously rebuilt on successful `main` test?

Reply:
I think we should start tagging instead :)